### PR TITLE
10.0 add sale shipping info helper

### PR DESCRIPTION
**Pull request by mourad-ehm**
This module, add a new computed field to sale order to make shipping info accessible for report or e-commerce.

### CODE REVIEW

File: sale_shipping_info_helper/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2013-2017 Camptocamp SA (Jacques-Etienne Baudoux)
+# © 2014-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
```
Comment (bealdav) [inline sale_shipping_info_helper/__manifest__.py]: really ?

File: sale_shipping_info_helper/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+{
+    'name': 'Sale shipping info helper',
+    'version': '10.0.1.0.0',
+    'category': 'Sales',
+    'license': 'AGPL-3',
+    'summary': 'Add shipping amounts on sale order',
+    'depends': ['sale', 'delivery'],
+    'author': 'akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
```
Comment (bealdav) [inline sale_shipping_info_helper/__manifest__.py]: It should be https://github.com/OCA/sale-workflow

**Review by bealdav — APPROVED** [review]
Except typo

File: sale_shipping_info_helper/models/sale_order.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            record.shipping_amount_untaxed = 0
+            record.shipping_amount_total = 0
+            record.shipping_amount_tax = 0
+            for line in record.order_line:
+                if line['is_delivery']:
+                    record.shipping_amount_untaxed = line['price_subtotal']
+                    record.shipping_amount_total = line['price_total']
+                    record.shipping_amount_tax = line['price_tax']
+        for key in ['amount_total', 'amount_untaxed', 'amount_tax']:
+            record['item_%s' % key] = record[key] - record['shipping_%s' % key]
```
Comment (hparfr) [inline sale_shipping_info_helper/models/sale_order.py]: out of scope variable

**Review by hparfr — CHANGES_REQUESTED** [review]

**Review by hparfr — APPROVED** [review]

**Comment by mourad-ehm** [comment]
ping @sebastienbeau

**Review by sebastienbeau — APPROVED** [review]
Good work, thanks

**Pull request merged**